### ISSUE TITLE
Could you support the case "every 2 minutes"?

### ISSUE THREAD

**Issue by sweety-wo** [RajanRastogi/angular-quartz-cron#2]
@RajanRastogi and @jacobscarter 
Now it only supports every minute,
but not supported  "**every N minutes**", 
for example `*/2 * * * *`, `*/3 * * * *`,..... `*/N * * * *`

I know **angular-quartz-cron** is base on "angular-cron-jobs" which not supported above issue.
But can you please let me know if you guys are going to support this feature in near future and if yes then how long i should wait for that feature update?

Thank you

**Comment by RajanRastogi** [comment]
@sweety-wo Thanks for opening the issue. It can be done, but I wanted a little more clarity on the requirement.

The thing with cron is that for operations such as every _N_ minutes, the result can be a little misleading. For example, if you schedule something to run every 40 minutes, the job will be triggered at every 40th minute of the hour, Instead of every 40 minutes. Let me know if my understanding is correct.

What we can do is provide a drop down with some common intervals (such as 1,2,3,4,5,10,15,20,30). Let me know if that works for you.

**Comment by sweety-wo** [comment]
@RajanRastogi 
Hi.. thank you for your quick reply i really appreciate. but what needed by me is "It should run at every 40 minutes instead every 40th minute of the hour". In short i have few job which should run on every 2 minute or so.. and that means it will fire 30 times in one hour.
Here expression will be like "0 0/2 \* 1/1 \* ? *"

Let me know if something still unclear for you..

Thanks.

**Comment by RajanRastogi** [comment]
Right, I understand that. However, what I am saying is that if we let the end user enter any number, it is error prone. The number needs to be a factor of 60. So in your case, 2 would work fine. However, if the user enters 40, it won't and would lead to a sort of an unexpected result.

So, in the default UI, should I limit the user choices to every - 1,2,3,4,5,10,15,20,30 minutes ?

**Comment by sweety-wo** [comment]
yeah..that will work perfectly for me..but how long that change should take? also just more question "are you going to support similar feature for every N second also?"

**Comment by RajanRastogi** [comment]
sure...can do..but even with seconds we will have the same limitation. I will take time off this weekend and implement them. I will share that branch with you (will update this issue). Would you be willing to validate it before I merge it into master?

**Comment by sweety-wo** [comment]
yeah you can just share your branch i will be checking it..and that will be really great if you can work on it this weekend. i really want it as soon as possible for you..

Thank you so much for your support.. :) :+1:

**Comment by jacobscarter** [comment]
@sweety-wo @RajanRastogi angular-cron-jobs has features like this keyed up for the next release, I haven't began the work yet, @RajanRastogi if you get this feature up this weekend I would like to also add it to my directive if you dont mind.

**Comment by jacobscarter** [comment]
Also @RajanRastogi angular-cron-job now supports multiple selections (e.g. `* * 4,5,6,7,8 3,4,5 *`)  So I don't know how much work it is to re-base your repo to mine but feel free to if you want to scoop up those new features.

**Comment by RajanRastogi** [comment]
@jacobscarter Those are some pretty neat features you got in the new release... I think its best we merge the code and maintain just one project. Would you mind if I could discuss something over email ?

Btw, was a little busy this weekend, so could not finish this feature.

**Comment by vincent-zurczak** [comment]
If you could merge your projects, that would be awesome! :innocent:

**Comment by jacobscarter** [comment]
@RajanRastogi once I finish this new feature push I'm working on now (hopefully) next few days, lets tackle merging these two!

**Comment by sweety-wo** [comment]
@RajanRastogi and @jacobscarter 

Hello guys,
Do you have any update for me?

**Comment by jacobscarter** [comment]
@sweety-wo I actually need to merge these two to get it to work on something I need at my day job, so I think I can make it happen, @RajanRastogi do you have time in case I have any questions for you?

**Comment by RajanRastogi** [comment]
@sweety-wo sorry was away for the past few weeks. Could not work on it.

@jacobscarter sure, let me know how you would like to go about it :)

**Comment by sweety-wo** [comment]
@RajanRastogi and @jacobscarter

Hello guys,
Do you have any update for me?

**Comment by sweety-wo** [comment]
@RajanRastogi and @jacobscarter
Hello guys,
Do you have any update for me?
I want to implement it in my project. So please reply me..

**Comment by RajanRastogi** [comment]
@sweety-wo sorry for the massive delays. We have started merging the two projects and the basic work is already done. You can now generate quartz compatible cron strings from angular-cron-jobs.
This is the next feature that will be implemented. Very sorry for the delay.
